config/proxy-example.json

```diff
@@ -49,12 +57,24 @@
           "backend": "http://localhost:9130"
         },
         {
-          "endpoint": "/ocs/v1.php/",
-          "backend": "http://localhost:9140"
+          "endpoint": "/ocs/",
+          "backend": "https://demo.owncloud.com",
+          "apacheHack": true
```

Comment:
Since we're using this value to take more than one action:

1. Write `req.Host`
2. Create a custom Transport that uses `HTTP/1.1`

And both actions are due to we're accessing an Apache server that is not configured to use `HTTP/2` nor to [preserve the hostname](http://httpd.apache.org/docs/2.2/mod/mod_proxy.html#proxypreservehost).

I propose renaming the key to something more explicit, such as `ocDefaultApache`, as it is the default apache config shipped with oc10. Again, I vouch for this instead of 2 flags because both preconditions need to exist in order for the proxy to work.

Needless to say such flag is unnecessary for the `reva` policy.

Reply:
maybe `apacheVHost`?

Reply:
`apacheVHost` tickles my fancy yeah

Reply:
going with `apache-vhost` after figuring out we need to use mapstructure and not json as the tag, because viper uses mapstructure to parse config files